staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -662,6 +662,7 @@ func AuthorizeClientBearerToken(loopback *restclient.Config, authn *Authenticati
 	}
 	if authn == nil || authz == nil {
 		// prevent nil pointer panic
```

Comment:
🤔

Reply:
IMO this function should be more strict about the inputs. The caller could check for nil instead of relying on implicit noop behavior.

Reply:
@liggitt, do you know why this function is so loose?

Reply:
it started in the apiserver, where we were trying to add in the loopback authn/authz. if no authn/authz was configured, this was supposed to be a no-op